elastic-stack/README.md

```diff
@@ -6,23 +6,23 @@
 
 ### DESCRIPTION
 
-Create links from SignalFx charts and alerts to any URL-addressable destination such as Kibana.
+Create links from Splunk Infrastructure Monitoring charts and alerts to any URL-addressable destination such as Kibana.
 
 The links appear in the Action menus adjacent to metric property values displayed in dashboards and alerts.
 
 The metric property values are used as query parameter and/or path variable values of the URL link.
 
 ### INSTALLATION
 
-Installation is actually configuring linking from property values on SignalFx charts and alerts to any URL-addressable destination. 
+Installation is actually configuring linking from property values on SignalFx charts and alerts to any URL-addressable destination.
 
 Read about the possible configurations [here](https://docs.signalfx.com/en/latest/dashboards/dashboard-links.html) and priority linking [here](https://docs.signalfx.com/en/latest/managing/data-links.html#local-and-global-data-links).
 
 To read about creating local data links, click [here](https://docs.signalfx.com/en/latest/managing/data-links.html#local-links).
 
 You must be an account administrator to configure new global links.
 
-To access Global Data Link configuration options page, click [here](/#/organization/YOUR_SIGNALFX_ORG_ID?selectedKeyValue=sf_section:globaldatalinks). 
+To access Global Data Link configuration options page, click [here](/#/organization/YOUR_SIGNALFX_ORG_ID?selectedKeyValue=sf_section:globaldatalinks).
```

Comment:
The link text doesn't conform to the Splunk Documentation Style Guide. See my previous comment.
The link target is not a valid URL.